digest/src/main/scala/akka/stream/alpakka/digest/DigestCalculator.scala

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright (C) 2016 Lightbend Inc. <http://www.lightbend.com>
+ */
+package akka.stream.alpakka.digest
+
+import akka.stream._
+import akka.stream.scaladsl.{ Flow, Keep, Sink }
+import akka.stream.stage._
+import akka.util.ByteString
+import akka.{ Done, NotUsed }
+
+import scala.concurrent.Future
+import scala.util.Success
+
+/**
+ * The DigestCalculator transforms/digests a stream of akka.util.ByteString to a
+ * DigestResult according to a given Algorithm
+ */
+object DigestCalculator {
+
+  def flow(algorithm: Algorithm): Flow[ByteString, DigestResult, NotUsed] =
```

Comment:
we decided to use scaladsl/javadsl packages, as in https://github.com/akka/alpakka/pull/19